Core/Source/MVVM/ViewModel.swift

```diff
@@ -65,12 +65,26 @@ public struct SecondaryActionInfo {
     public let imageName: String?
 }
 
+/// Describes the parent item of a list of SecondaryActions.
+public struct SubactionsRootInfo {
+
+    public init(title: String, enabled: Bool = true, imageName: String? = nil) {
```

Comment:
Could we just roll the `[SecondaryAction]` property into this too so it can just be `SecondaryAction.SubactionsRootInfo(SubactionsRootInfo)`

Reply:
Also this wasn't introduced in this diff, but `Subactions`/`SubactionsInfo` doesn't really seem right. 

I know it sort of matches with `NSMenu.setSubmenu`... but just reads pretty funny to me. 

What do you think about using "nested" instead like `case nested(NestedActionsInfo)`?

Reply:
Ooh I like that